ui/src/userProfile.ts

```diff
@@ -24,8 +26,22 @@ export const fetchProjects = (urn: string): void => {
 };
 
 export const fetchUser = (urn: string): void => {
-  identity
-    .fetch(urn)
+  proxy.client
+    .remoteIdentityGet(urn)
     .then(userStore.success)
     .catch(err => userStore.error(error.fromUnknown(err)));
 };
+
+export const openUserProfile = (
+  userUrn: string,
+  sessionIdentityUrn: string
+): void => {
+  if (userUrn === sessionIdentityUrn) {
```

Comment:
As an alternative, we could get the session identity URN from the `ui/src/session` module. The advantage would be that we don’t need to obtain it first every time we want to call this function.

Reply:
I thought of this as well. But then I realised that we set the session via a context in a top-level component, however the modal is outside of that component, so when we try to get the session via `getUnsealedFromContext` it fails.

I think if we want to solve this, we'll have to refactor how we pass the session around. It might make sense to just have a global session store instead of the context. This would be a bigger rewrite though.